### ISSUE TITLE
Error creating looker_project_git_repo

### ISSUE THREAD

**Issue by DanialZohairen** [devoteamgcloud/terraform-provider-looker#27]
I'm trying to create a project by cloning an existing repo using this resource. It gives out an error 404:
![image](https://user-images.githubusercontent.com/98127526/208959014-c985205d-ef4e-42e3-9420-245d242e0e77.png)

What I did to recreate this:
![image](https://user-images.githubusercontent.com/98127526/208959354-2ee0ff68-5d74-438e-a2f7-c10a41dba397.png)

**Comment by Nabil-Lahssini** [comment]
Hi Danial, it seems like Looker doesn't find your project. Do you have sufficient permission with your account to retrieve the project? Is the name of the project completely correct?

**Comment by DanialZohairen** [comment]
I was under the impression that this resource would create a project instead of looking for an existing one. If this isnt what I think it is, is there a way to create a project from a clone repo?

**Comment by Nabil-Lahssini** [comment]
The first step is to create a project using the "looker_project" resource. Then, you can bind your project to an existing git repository using the "looker_project_git_repo" resource.
Don't forget to add username and password as parameters, for more information about the needed parameters refer to the documentation at [this page](https://registry.terraform.io/providers/devoteamgcloud/looker/latest/docs/resources/project_git_repo).

**Comment by DanialZohairen** [comment]
I created a blank project using "looker_project". But I still cant bind/link the repo to the created project, as it gives the same error. I tried to link using SSH instead of HTTPS to avoid using git username and password:
![image](https://user-images.githubusercontent.com/98127526/209004254-0dde14e4-ae26-4499-b610-04423819dc3f.png)

Looker still can't seem to find the resource I want to create.
